controllers/application/service_monitor.go

```diff
@@ -6,11 +6,19 @@ import (
 	"github.com/kartverket/skiperator/pkg/util"
 	pov1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	ctrlutil "sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+var (
+	IstioMetricsPortNumber = intstr.FromInt(15020)
+	IstioMetricsPortName   = intstr.FromString("istio-metrics")
+	IstioMetricsPath       = "/stats/prometheus"
+)
+
+// TODO: Restore on delete
```

Comment:
TODO for later or this PR? :)

Reply:
Dangling `TODO`, already fixed [here](https://github.com/kartverket/skiperator/commit/480b89a1cb51945ab2753c52f8321b38a7a7023a#diff-34341ce42c94650ff2ae6969df83c3eb600a1cae903f85df928954ecc2b9bbf0R73)